### PR TITLE
More detail added to the descriptions.

### PR DESCRIPTION
**Pull request by axayjo**
Changes made to description strings.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/gallery.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the gallery."
+            "description": "The name of the Shared Image Gallery. The allowed characters are alphabets and numbers with dots, dashes, and periods allowed in the middle. The maximum length is 80 characters as limited by Constants.MaxResourceNameLength for any type of resource created in Azure."
```
Comment (singhkays) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/gallery.json]: Don't need this "as limited by Constants.MaxResourceNameLength for any type of resource created in Azure." as people don't know what Constants.MaxResourceNameLength is

File: specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/gallery.json [hunk 2]
```diff
@@ -213,14 +213,14 @@
...
-            "description": "The name of the gallery."
+            "description": "The name of the Shared Image Gallery in which the Image Definition is to be created."
           },
           {
             "name": "galleryImageName",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the gallery image."
+            "description": "The name of the gallery Image Definition to be created or updated. The allowed characters are alphabets and numbers with dots, dashes, and periods allowed in the middle. The maximum length is 80 characters as limited by Constants.MaxResourceNameLength for any type of resource created in Azure."
```
Comment (singhkays) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/gallery.json]: Same here as above re: Constats

**Review by singhkays — COMMENTED** [review]
We should also add to the descriptions which of the fields is updatable

**Comment by singhkays** [comment]
lgtm

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**